packages/tool/site/client/tests/components/prescription-summary/prescriptionSummaryView.spec.tsx

```diff
@@ -4,7 +4,7 @@ import {summaryPrescription} from "./props"
 import pretty from "pretty"
 import {render} from "@testing-library/react"
 
-test("Renders correctly", async () => {
+test.skip("Renders correctly", async () => {
```

Comment:
Ideally we wouldn't be skipping a previously passing test - chat to Matthew Popat to see if this can be resolved